operator/api/vectorized/v1alpha1/cluster_types.go

```diff
@@ -166,6 +166,12 @@ type ClusterSpec struct {
 	// By default if Replicas is 3 or more and redpanda.default_topic_partitions is not set
 	// default webhook is setting redpanda.default_topic_partitions to 3.
 	AdditionalConfiguration map[string]string `json:"additionalConfiguration,omitempty"`
+	// Cluster configuration values may also be held here. A `keyName` entry in this
```

Comment:
Is the distinction between `keyName` and `keyname` intentional / important here?

Reply:
It's not, fixed.